stack.yaml

```diff
@@ -74,28 +74,28 @@ extra-deps:
 - gray-code-0.3.1
 
 - git: https://github.com/well-typed/cborg
-  commit: 42a83192749774268337258f4f94c97584b80ca6
+  commit: 8e12924f5b34ac9727a481ef455fc5629f42be8c
```

Comment:
This is going backwards by 2 commits. Was this intended?

Reply:
Hmm. No it isn't. We do want the pretty print fix.